fancy_collections/formatting.py

```diff
@@ -17,9 +17,10 @@ def __init__(
         max_rows: int | None = None,
         min_rows: int | None = None,
         show_df_column_names: bool = True,
+        max_colwidth: int = 50
     ):
         self.show_df_column_names = show_df_column_names
-        self._trunc_options = dict(max_rows=max_rows, min_rows=min_rows)
+        self._trunc_options = dict(max_rows=max_rows, min_rows=min_rows, max_colwidth=max_colwidth)
```

Comment:
unfortunatly this will not work with `pd.Series.to_string`, which we also use with `_trunc_options`.. 

i can fix this after lunch and merge it.

Reply:
FYI: `trunc_options` is intended for vertical truncation only, but i must admit that this is badly documented ;)